v3.1/virus-check.pl

```diff
@@ -19,8 +19,8 @@
 use warnings;
 use POSIX qw(strftime);
 
-my $do_log = 1;
-my $logfile = "/tmp/virus-check.log";
+my $do_log = $ENV{"CLAMD_DEBUG_LOG"} eq "on" ? 1 : 0;
+my $logfile = "/dev/stdout";
```

Comment:
I'd suggest to stick to the (unwritten?) PHP and Bash rule for strings: Use double quotes only if you want values inside to be interpreted (or if you have single quotes inside the string and escaping would make it less readable).

More reading:

- [Perl: How to access shell environment variables through Perl associative arrays](https://alvinalexander.com/perl/edu/articles/pl020002)
- [Contrasting Single and Double Quotes in Perl](https://www.dummies.com/programming/perl/contrasting-single-and-double-quotes-in-perl/)

Reply:
I'll stick to double quotes as it's used everywhere else in the script already.